### ISSUE TITLE
Formal German language conflict with AdSense tables

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#3285]
## Bug Description

When the formal German language is applied as the primary language the AdSense dashboard don't display, in either the primary Site Kit dashboard or in the AdSense tab. 

**Primary Site Kit dashboard:**

Once accessed a "Data error in AdSense Invalid Value" message appears
 
Console errors
```
JQMIGRATE: Migrate is installed, version 3.3.2
/wp-json/google-site-kit/v1/modules/adsense/data/earnings?startDate=2021-04-06&endDate=2021-05-03&metrics%5B0%5D=EARNINGS&metrics%5B1%5D=PAGE_VIEWS_RPM&metrics%5B2%5D=IMPRESSIONS&dimensions%5B0%5D=DATE&_locale=user:1 Failed to load resource: the server responded with a status of 400 ()
/wp-json/google-site-kit/v1/modules/adsense/data/earnings?startDate=2021-04-06&endDate=2021-05-03&metrics=EARNINGS&_locale=user:1 Failed to load resource: the server responded with a status of 400 ()
2googlesitekit-api.74da2dc9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
(anonymous) @ googlesitekit-api.74da2dc9d2273da62d71.js:3
/wp-json/google-site-kit/v1/modules/adsense/data/earnings?startDate=2021-04-06&endDate=2021-05-03&metrics%5B0%5D=EARNINGS&metrics%5B1%5D=PAGE_VIEWS_RPM&metrics%5B2%5D=IMPRESSIONS&_locale=user:1 Failed to load resource: the server responded with a status of 400 ()
googlesitekit-api.74da2dc9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
(anonymous) @ googlesitekit-api.74da2dc9d2273da62d71.js:3
/wp-json/google-site-kit/v1/modules/adsense/data/earnings?startDate=2021-03-09&endDate=2021-04-05&metrics%5B0%5D=EARNINGS&metrics%5B1%5D=PAGE_VIEWS_RPM&metrics%5B2%5D=IMPRESSIONS&_locale=user:1 Failed to load resource: the server responded with a status of 400 ()
googlesitekit-api.74da2dc9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
(anonymous) @ googlesitekit-api.74da2dc9d2273da62d71.js:3
```

![image](https://user-images.githubusercontent.com/41326532/117053454-e3ed6500-ad10-11eb-9797-b67466ca9dd0.png)

**AdSense dashboard:**

In addition to the primary Site Kit dashboard when accessing the AdSense tab no data is available, with the below appearing:

![image](https://user-images.githubusercontent.com/41326532/117054072-a50bdf00-ad11-11eb-90ea-462d674096f9.png)

Unexpanded console errors below (for the AdSense tab). Login to the shared AdSense testing site for full details (or follow steps to reproduce)

```
JQMIGRATE: Migrate is installed, version 3.3.2
googlesitekit-vendor…fe8c6341fea5bc.js:1 GET https://oi.ie/wp-json/google-site-kit/v1/modules/adsense/data/earnings?metr…5B3%5D=PAGE_VIEWS_CTR&startDate=2021-04-07&endDate=2021-05-04&_locale=user 400
googlesitekit-api.74…c9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
googlesitekit-vendor…fe8c6341fea5bc.js:1 GET https://oi.ie/wp-json/google-site-kit/v1/modules/adsense/data/earnings?metr…5B3%5D=PAGE_VIEWS_CTR&startDate=2021-03-10&endDate=2021-04-06&_locale=user 400
googlesitekit-api.74…c9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
googlesitekit-vendor…fe8c6341fea5bc.js:1 GET https://oi.ie/wp-json/google-site-kit/v1/modules/adsense/data/earnings?dime…5B3%5D=PAGE_VIEWS_CTR&startDate=2021-03-10&endDate=2021-04-06&_locale=user 400
googlesitekit-api.74…c9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
googlesitekit-vendor…fe8c6341fea5bc.js:1 GET https://oi.ie/wp-json/google-site-kit/v1/modules/adsense/data/earnings?dime…5B3%5D=PAGE_VIEWS_CTR&startDate=2021-04-07&endDate=2021-05-04&_locale=user 400
googlesitekit-vendor…fe8c6341fea5bc.js:1 GET https://oi.ie/wp-json/google-site-kit/v1/modules/adsense/data/earnings?startDate=2021-04-06&endDate=2021-05-03&metrics=EARNINGS&_locale=user 400
googlesitekit-api.74…c9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
googlesitekit-api.74…c9d2273da62d71.js:3 Google Site Kit API Error method:GET datapoint:earnings type:modules identifier:adsense error:"Invalid Value"
```

## Steps to reproduce

1. Login to a site with AdSense traffic
2. Switch the language to German (Formal)
3. Visit the primary Site Kit dashboard - invalid data message appears
4. Visit the AdSense tab - AdSense gathering data notice appears

## Additional Context

 - Site Kit 1.31.0
 - [Original support topic](https://wordpress.org/support/topic/data-error-in-adsense/#post-14398681)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

### Visual Regression Changes

* 

## QA Brief

* 

## Changelog entry

*

**Comment by jamesozzie** [comment]
Reproducible in support, assigning to L2 for review.

**Comment by Hazlitte** [comment]
Hi @jamesozzie, This bug is being caused by the `Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.` error which we worked on in issue [2280](https://github.com/google/site-kit-wp/issues/2280).  If the solution applied to the components referenced in that issue is also applied to the `ReportTable` component I suspect this bug will be resolved.

**Comment by bethanylang** [comment]
@jamesozzie @abdullah1908 Per @Hazlitte's comment above, can you please test to see if this has been resolved with the fix from #2280? If so, let's get this closed and reply to the affected users. Thanks!

**Comment by jamesozzie** [comment]
Confirming all is good with formal German. Thank you @Hazlitte 

Closing while also following up with the user who initially reported in the support forums.
